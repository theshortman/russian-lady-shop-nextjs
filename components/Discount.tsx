import { FC } from "react";

interface DiscountProps {
  value: number;
}

const Discount: FC<DiscountProps> = ({ value }) => {
  return <div className="badge bg-danger">-{value}%</div>;
};

export default Discount;
