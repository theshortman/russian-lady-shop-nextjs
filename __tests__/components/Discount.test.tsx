import { render } from "@testing-library/react";
import Discount from "@/components/Discount";

test("renders Discount unchanged", () => {
  const { container } = render(<Discount value={30} />);
  expect(container).toMatchSnapshot();
});
