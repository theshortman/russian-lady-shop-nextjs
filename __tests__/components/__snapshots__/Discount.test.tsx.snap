// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Discount unchanged 1`] = `
<div>
  <div
    class="badge bg-danger"
  >
    -
    30
    %
  </div>
</div>
`;
